packages/now-client/src/check-deployment-status.ts

```diff
@@ -9,12 +9,7 @@ import {
   isAliasError,
 } from './utils/ready-state';
 import { createDebug } from './utils';
-import {
-  Dictionary,
-  Deployment,
-  NowClientOptions,
-  DeploymentBuild,
-} from './types';
+import { Deployment, NowClientOptions, DeploymentBuild } from './types';
 
 interface DeploymentStatus {
   type: string;
```

Comment:
```suggestion
  type: DeploymentEventType;
```

I think this needs to be updated

Reply:
👍